README.md

```diff
@@ -1,24 +1,136 @@
-# README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet) ![Ruby on Rails](https://img.shields.io/badge/-Ruby%20on%20rails-blueviolet)
+<img src="https://img.shields.io/badge/rails-%23CC0000.svg?style=for-the-badge&logo=ruby-on-rails&logoColor=white" alt="ruby" height="auto" />
 
-Things you may want to cover:
+<div align="center">
+  <h1><b>Nazmaly</b></h1>
+  <a name='readme-top'></a>
+</div>
 
-* Ruby version
+# 📗 Table of Contents
 
-* System dependencies
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Loom Video](#live-demo)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* Configuration
+# 📖 Blog Rails project <a name="about-project"></a>
 
-* Database creation
+> Welcome to Nazmaly, the ultimate money transaction app for professionals. With Nazmaly, you can easily and securely manage all your financial transactions in one place. Our user-friendly interface and advanced security features make it simple and safe to send and receive payments, view transaction history, and manage your digital wallet.
 
-* Database initialization
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+### Tech Stack <a name="tech-stack"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+  </ul>
 
-* Deployment instructions
+</details>
 
-* ...
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Add comments**
+- **Like Posts**
+
+<p align="right">([back to top](#readme-top))</p>
+
+## 🚀 Loom Video <a name="live-demo"></a>
+
```

Comment:
- [ ] Your app is apparently not deployed, as live demo link cannot be found in the ReadMe. Kindly ensure to/that:
  * deploys the application using Render, Heroku or similar for Rails app.
  * a link to the deployed project is present in the ReadMe.